docs/configuration.md

```diff
@@ -162,3 +146,22 @@ spec:
   featureGates:
     deployTektonTaskResources: true
 ```
+
+### `deployVmConsoleProxy`
+
+Set the `deployVmConsoleProxy` feature gate to true to allow the operator
+to deploy VM console proxy resources.
+
+Resources that provides access to the VNC console of a Kubevirt VM
```

Comment:
You may add that it enables access to VMs without access to the cluster's API.

Reply:
Done.